docs/develop/worker-performance.mdx

```diff
@@ -36,10 +37,176 @@ The Workflow Cache is created and shared between all the workers. It's designed
+  Activity Tasks represent units of work within a larger business process.
+  Each Activity Task contains the context required by an Activity Definition.
+  Workers poll Activity Tasks from the Activity Task Queue and use them to initiate Activity Executions.
+- **Workflow Task Queue**: A queue that holds Workflow Tasks.
+  Workflow Tasks contain the context needed by a Workflow Definition.
+  Workers poll Workflow Tasks from the Workflow Task Queue and use them to initiate Workflow Executions.
+- **Nexus Task Queue** ([Public Preview](/evaluate/development-production-features/release-stages#public-preview)): A queue that holds Nexus Tasks.
+  Nexus Tasks are used for units of work passed between Namespaces.
+  Workers configured with the Nexus Service poll the Nexus Task Queue for available tasks and handle them by initiating Workflow Executions in the target Namespace.
+  Each Nexus Task contains the context required by the target Workflow Definition.
+
+Each Task Queue type provides unaggregated metrics.
+
+### Task Queue metrics {#task-queue-metrics-list}
+
+The Temporal Service reports information separately for each Task Queue type (not aggregated).
+Use the following metrics to retrieve detailed information about Task Queue health and performance.
+Available metrics include:
+
+- [`ApproximateBacklogCount`](#ApproximateBacklogCount)
+- [`ApproximateBacklogAge`](#ApproximateBacklogCount)
+- [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate)
+- [`BacklogIncreaseRate`](#BacklogIncreaseRate) (derived from [`TasksAddRate`](#TasksAddRate-and-TasksDispatchRate) and [`TasksDispatchRate`](#TasksAddRate-and-TasksDispatchRate))
+
+#### `ApproximateBacklogCount` {#ApproximateBacklogCount}
+
+Represents the approximate count of Tasks currently backlogged in this Task Queue.
+The number may include expired Tasks as well as active Tasks, but it will eventually converge to the correct count over time.
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide partial information due to performance optimizations.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values for this metric.
+Since Tasks remain valid for only a few seconds in Sticky Queues, this inaccuracy diminishes over time, especially as the backlog grows.
+:::
+
+
+#### `ApproximateBacklogAge` {#ApproximateBacklogAge}
+
+Returns the approximate age of the oldest Task in the backlog.
+The age is based on the creation time of the Task at the head of the queue.
+
+You can rely on this count when making scaling decisions.
+
+:::info Metric Accuracy
+
+Workflow Task Queue types provide partial information due to performance optimizations.
+Tasks sent to [Sticky](https://docs.temporal.io/workers#sticky-execution) queues are not included in the returned values.
```

Comment:
Do you mean Sticky Execution instead of Sticky Queue? I went on that URL, and don't see the specific 'sticky queue' verbage used

Reply:
> A Sticky Execution is when a Worker Entity caches the Workflow in memory and creates a dedicated Task Queue to listen on.

The dedicated Queue is the Sticky Queue